Gemfile

```diff
@@ -87,8 +87,8 @@ gem "chartkick"
 # elasticsearch we use
 # See https://github.com/elastic/elasticsearch-ruby#compatibility
 gem "elasticsearch-api", "~> 2.0.0"
-gem "elasticsearch-model"
-gem "elasticsearch-rails"
+gem "elasticsearch-model", ">= 5.0.0"
+gem "elasticsearch-rails", ">= 5.0.0"
```

Comment:
When we release, we'll need to do the shuffle to get on the new elastic search on heroku.